corelibs/libgraph.md

```diff
@@ -1,36 +1,352 @@
+
+  - _`queue`_ - graphics task queue, should be chosen from the `graph_queue_t` enum, where are following options: `GRAPH_QUEUE_HIGH` - high priority queue, `GRAPH_QUEUE_LOW` - low priority queue, `GRAPH_QUEUE_BOTH` - both queues are chosen, `GRAPH_QUEUE_DEFAULT` - default queue.
+
+
+- `int graph_rect(graph_t *graph, unsigned int x, unsigned int y, unsigned int dx, unsigned int dy, unsigned int color, graph_queue_t queue)` </br>
+  Draws a rectangle. Arguments are similar to those in graph_line function. A drawn rectangle will be filled with a specified color.
+
+- `int graph_fill(graph_t *graph, unsigned int x, unsigned int y, unsigned int color, graph_fill_t type, graph_queue_t queue)` </br>
+  Fills polygon limited by drawn lines (when type is `GRAPH_FILL_FLOOD`), or limited by screen boundaries (`GRAPH_FILL_BOUND` type).
+
+- `int graph_print(graph_t *graph, const graph_font_t *font, const char *text, unsigned int x, unsigned int y, unsigned char dx, unsigned char dy, unsigned int color, graph_queue_t queue)` </br>
+  Prints text pointed by the _`text`_ argument. Font data should be passed to `graph_font_t` structure. The example is stored in `gfx` directory in [phoenix-rtos-tests](https://github.com/phoenix-rtos/phoenix-rtos-tests.git) repository (`font.h` file). The remaining arguments are similar to those from functions above.
+
+- `int graph_move(graph_t *graph, unsigned int x, unsigned int y, unsigned int dx, unsigned int dy, int mx, int my, graph_queue_t queue)` </br>
+  Moves data in the range specified by _`x`_, _`y`_, _`dx`_, _`dy`_ arguments to the following point: (_`x`_ + _`mx`_, _`y`_ + _`my`_).
+
+- `graph_copy(graph_t *graph, const void *src, void *dst, unsigned int dx, unsigned int dy, unsigned int srcspan, unsigned int dstspan, graph_queue_t queue)` </br>
+  Copies a bitmap pointed by the _`src`_ argument into bitmap pointed by the _`dst`_ argument. The area, which is copied is limited by a rectangle with _`dx`_ and _`dy`_ dimensions. There should also be specified span arguments, which is the total width of a source/destination bitmap multiplied by a color depth. When copying some part of a bitmap, _`src`_ should point to the proper element, same with destination buffer. By default, the following bitmap's format is applied: {B, G, R, A, B, G, R, A}, so when color depth equals 4 in one row of array are stored 2 pixels. For cirrus graphics adapter (default), alpha (represented by `A`) isn't supported, so opacity is always 100%.
+
+- `int graph_colorset(graph_t *graph, const unsigned char *colors, unsigned char first, unsigned char last)` </br>
+  Sets a color palette for the specified graphics adapter context. Used only for modes with 8-bit color depth. A color map should be passed in _`cmap`_ argument. The range of changing colors is set by passing _`first`_ and _`last`_ arguments. If a set color palette's size is lower than a default one, remaining colors are the same.
```

Comment:
I'd change the first sentence to:
Sets a color palette used for 8-bit indexed color mode. Not supported for direct color modes.